packages/assets-controllers/src/RatesController/types.ts

```diff
@@ -0,0 +1,119 @@
+import type {
+  RestrictedControllerMessenger,
+  ControllerGetStateAction,
+  ControllerStateChangeEvent,
+} from '@metamask/base-controller';
+
+import type { fetchMultiExchangeRate as defaultFetchExchangeRate } from '../crypto-compare-service';
+import type { name as ratesControllerName } from './RatesController';
+
+/**
+ * Represents the conversion rates from one currency to others.
+ * Each key is a string representing the cryptocurrency code (e.g., "BTC", "SOL"),
+ * and its value is either a number representing the conversion rate to that currency,
+ * or `null` if the conversion rate is not available.
+ */
+export type Rate = {
+  conversionDate: number;
+  conversionRate: string;
```

Comment:
Also, adds a comment to explain why we use a `string` to represent the rate maybe?

Reply:
Done in [cda3a20](https://github.com/MetaMask/core/pull/4242/commits/cda3a208d26f50a5798425e9ee8409da28f7dba7)